src/main/resources/changesets/failure_table.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="chrisl" id="failure_table">
+        <createTable tableName="FAILURE">
+            <column autoIncrement="true" name="FAILURE_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_FAILURE"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="EXECUTION_ID" type="INT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="FAILURE" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="TIMESTAMP" type="DATETIME">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Now `TIMESTAMP` *is* a MySQL reserved word, and actually it might be the data type you want to use here.  On Oracle it matters a lot, I don't know MySQL so well.